Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -41,23 +41,24 @@ class Navigator {
         }
     }
     
-    let historyRecorder: HistoryRecorderHandle
+    private(set) var historyRecorder: HistoryRecorderHandle
```

Comment:
Currently these changes are useless, but it's a fallout of it's previous base PR. This will be utilised with Navigator Version switching